project1/css/index.css

```diff
@@ -62,3 +62,103 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+a{
+	color:gray;
+	text-decoration:none;
+}
+ul{
+	position:relative;
+	top:8px;
+}
+li {
```

Comment:
Same thing here- make sure you are either spacing or not spacing _all_ of the target elements and their respective curly braces.